app/src/marketplace/components/ExpirationCounter/index.jsx

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import React, { useState } from 'react'
+import cx from 'classnames'
+
+import useInterval from '$shared/hooks/useInterval'
+
+import styles from './expirationCounter.pcss'
+
+type Props = {
+    expiresAt: Date,
+    className?: string,
+}
+
+const DAY_SECONDS = 24 * 60 * 60
+
+const getExpirationString = (secondsLeft: number): string => {
+    if (secondsLeft < DAY_SECONDS) {
+        const date = new Date(0)
+        date.setTime(secondsLeft * 1000)
+        return date.toISOString().substr(11, 8) // hh:mm:ss
+    }
+
+    const days = parseInt(secondsLeft / DAY_SECONDS, 10)
+    return `${days} day${days > 1 ? 's' : ''}`
+}
+
+const Error = ({ expiresAt, className }: Props) => {
```

Comment:
Probably shouldn't call this `Error` in case you ever actually want to throw a real `Error` e.g. `throw new Error('oops')`, plus it'd be easy to miss the problem unless the code actually goes down that exception path.

Reply:
i.e. perhaps call it `ExpirationCounter`?